longevity_test.py

```diff
@@ -60,16 +62,15 @@ def test_custom_time(self):
             stress_queue.append(self.run_stress_thread(**params))
             if 'profile' in params:
                 del params['profile']
+        if not prepare_write_cmd:
+            self.db_cluster.start_nemesis(interval=self.params.get('nemesis_interval'))
 
-        self.db_cluster.wait_total_space_used_per_node()
```

Comment:
@amoskong, why did you remove the wait for total space?
We need it in case there is no prepare_write_cmd...

Reply:
@roydahan I will add it back.